Added ALT Tag for getting-started/implementation-plan.rst (#5593)
mattermost/docs: source/getting-started/implementation-plan.rst

The following diagram is a high-level illustration of the Mattermost platform and how its various components interact with each other within a network:

.. image:: ../images/network_diagram.png

>    :alt: Matternost network diagram that illustrates about Mattermost platform and interaction of its various components with each other

Reviewer: ```suggestion
   :alt: Mattermost network diagram shows how the various system components of the Mattermost platform interact with each other within a network deployment.
```
